specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/examples/GetUsagesInLocation.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "location": "West US",
+    "quotaType": "regional",
+    "api-version": "2023-01-01"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "value": [
```

Comment:
Why doesn't the payload of each 'usage' response contained in the list look like an ARM resource?

If you want a non-resource looking response payload, I am under the impression it has to be a POST method. [except for some special ARM APIS that don't follow the resource model conventions - this isn't one of those I presume?]

Reply:
I am not returning an ARM resource but rather an object with information about the usage in a given region. Other RPs have implemented this method as a GET method already: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/labservices/resource-manager/Microsoft.LabServices/stable/2023-06-07/Usages.json

https://learn.microsoft.com/en-us/rest/api/compute/usage/list?tabs=HTTP

Reply:
@rkmanda Right... sounds like this is becoming one of those instances of APIs where ARM guidelines weren't followed, and then it became a pattern... or am I just imagining some rules on GET responses that don't apply universally?